### PR TITLE
Hide classroom content from visitors on welcome page

### PR DESCRIPTION
**Pull request by tschaffer1618**
Concerns: Let me know your thoughts on possibly moving the 2 private helper methods I made in the welcome controller into a welcome facade.

Functionality: I made a 2 tests (one user and one visitor) with 2 tutorials each (one tutorial is classroom content and the other is not). The visitor test verifies that only the non-classroom content tutorial is shown on the welcome page, while the user test verifies that both tutorials are displayed.

### CODE REVIEW

**Comment by hsmitha26** [comment]
Pair review: Yes, I agree that creating a façade works well to implement:
1. skinny controller
2. encapsulation
Abstraction using private methods looks great.  Nice work!

**Comment by tschaffer1618** [comment]
Created a welcome facade per the discussion above. Tests still pass.

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -2,10 +2,11 @@
```
Comment (hsmitha26) [inline app/controllers/welcome_controller.rb:2]: Beautiful, skinny controller!

**Review by hsmitha26 — APPROVED** [review]
Looks great.  Thank you.

**Pull request merged**